pb_decode.c

```diff
@@ -1431,6 +1431,9 @@ static bool checkreturn pb_dec_bytes(pb_istream_t *stream, const pb_field_t *fie
 #ifndef PB_ENABLE_MALLOC
         PB_RETURN_ERROR(stream, "no malloc support");
 #else
+        if (stream->bytes_left < size)
```

Comment:
This check is already implemented in `pb_read()`, which is called later by this func, but after allocation happens, so check in pb_read is too late IIUC. 
Since this is a backport (from latest version of Nanopb?), do you know if there are other Nanopb security-related fixes that we should backport into our customized version?

Reply:
Replied offline.